packages/devtools_app/lib/src/memory/memory_protocol.dart

```diff
@@ -117,6 +120,10 @@ class MemoryTracker {
       adbMemoryInfo = AdbMemoryInfo.empty();
     }
 
+    // Query the engine's rasterCache estimate.
+    rasterCache = await _fetchRasterCacheInfo();
+    logger.log('$rasterCache');
```

Comment:
probably avoid logging this.

Reply:
I've added a TODO to remove logging, added for Yuqian to help track issues values not changing.